### ISSUE TITLE
Option to disable block highlighting

### ISSUE THREAD

**Issue by atulmy** [warpdotdev/warp#3109]
```Text
Option to disable block highlighting when selected.
```

### How important is this feature to you?

5 (very important)

### Warp Internal (ignore) - linear-label:770f6576-d6c0-4e4f-a259-fc64b5156087

**Comment by dannyneira** [comment]
Thanks for this feature request! 

Please add a :+1: to the original post at the top to signal that you want this feature, and then hit the subscribe button if you'd like to be notified.

**Comment by atulmy** [comment]
<img width="644" alt="Screenshot 2023-05-25 at 11 50 55 AM" src="https://github.com/warpdotdev/Warp/assets/685786/50b62fb4-b5ab-4b16-af01-d16c5c315c69">

I was referring to the pink translucent highlight along with the border which feels unnecessary and isn't a behaviour in regular terminal apps.

Having a toggle in setting would help.

**Comment by jesseduffield** [comment]
Ah, I didn't realise the highlighting was affected by the theme (mine is blue). In that case, yep we're talking about the same thing.

**Comment by muscaiu** [comment]
Any news on this?

**Comment by atulmy** [comment]
I gave up on warp and already reverted to default macos terminal with github.com/ohmyzsh/ohmyzsh

**Comment by namoshizun** [comment]
+1, the block highlighting is a total distraction to me and too looking for a way to turn it off

**Comment by KNOCKInc** [comment]
+1

**Comment by mikelpom** [comment]
+1

**Comment by ratbekn** [comment]
@atulmy can you please reopen the issue?

**Comment by finografic** [comment]
+1

**Comment by dannyneira** [comment]
Hey folks, reopening to track interest in this feature request to disable block highlights. Also we've recently disabled block highlighting when selecting text in Warp which may help some with not obscuring the text while selecting.

![CleanShot 2024-07-08 at 17 09 05](https://github.com/warpdotdev/Warp/assets/16809145/f821b8fc-e498-43af-ba2f-5d3c6e415791)

**Comment by oguzgelal** [comment]
+1

**Comment by ratbekn** [comment]
+1

**Comment by junkystu** [comment]
+1

**Comment by liunice** [comment]
+1000

**Comment by mrponzo** [comment]
yes, please

**Comment by demyxco** [comment]
This "feature" is so annoying! I'm trying to redo a command and it defaults to highlighting a block of previous output. Please please please code the feature to disable this.

**Comment by caiobsilva** [comment]
Please, please, please let me disable this. This has been so frustrating

**Comment by hwennnn** [comment]
+10000000

**Comment by lucidNTR** [comment]
after login requirements have been removed, this is the main blocker to use warp full time. the contrast with this highlight is so bad and it is very distracting. i just assumed it would be a simple select theme and modify a color in the scheme but there is just an option to do automatic changes based on a picture, very disheartening.

**Comment by demyxco** [comment]
This was closed but block highlighting is still enabled even after turning off all the checkboxes in Settings > Appearance > Blocks. There's nothing in the Features panel either. Changelog shows nothing for version `v0.2025.01.08.08.02.stable_04`. Am I missing something?

**Comment by lucidNTR** [comment]
> This was closed but block highlighting is still enabled even after turning off all the checkboxes in Settings > Appearance > Blocks. There's nothing in the Features panel either. Changelog shows nothing for version `v0.2025.01.08.08.02.stable_04`. Am I missing something?

yep, i was in the same situation, they improved the contrast slightly but no way to disable block highlighting or adjust the theme to something less annoying. i wonder who the warp devs dont see this?

**Comment by Refrigetwo** [comment]
+1

**Comment by ethanmchls** [comment]
+1

**Comment by frankmarineau** [comment]
+1, this is an absolute must guys..

**Comment by FelipeDanielH** [comment]
+1

**Comment by lucidNTR** [comment]
@atulmy please reopen this!

**Comment by sauron918** [comment]
+1
Leave border, make color customizable. But please make the background color optional. It's very distracting

**Comment by MostHated** [comment]
I very much hate the block highlighting.

**Comment by AGontcharov** [comment]
+1

**Comment by tmorgan497** [comment]
+1

**Comment by suncom-todd-chambery** [comment]
+1

**Comment by yashcrest** [comment]
what an absolute joke has warp been? why is this issue simply abandoned by warp? why is there no update on this?

**Comment by 0xFlo** [comment]
It still selects the entire block on click. making it impossible to use something like claude code. Good bye warp

**Comment by dyl-young-mohara** [comment]
+1

**Comment by CaglayanDokme** [comment]
@atulmy Unfortunately, the feature hasn't been implemented yet. Can you reopen the issue?

**Comment by MostHated** [comment]
Please do reopen it. I absolutely hate the block highlighting.

**Comment by aleesaan** [comment]
I've recently started using warp and love it, except this block highlighting thing 😓 why can't we make it configurable via a setting? Looking at this issue I didn't understand the answer.

**Comment by xcrap** [comment]
> Yeah I'm considering switching from Warp just because of this

That was actually the reason i left and never returned, it seems this haven't changed lol.

**Comment by atulmy** [comment]
Very strange even after so many fellow users complaining about this issue (which is very quick fix), still there seems no solution yet. Sad.
